components/TopBar.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-app-bar flat color="transparent" class="ma-1" height="60" :elevation="0">
+    <v-row class="pa-1 ma-1 align-center">
+      <v-img
+        src="/assets/img/logo.png"
```

Comment:
@JulienChampagnol you used vite import in the website, what is the best way?

Reply:
<template>
<v-img :src="image" />
</ template>
<script setup>
import image from "@/assets/image.png"
</ script>

Reply:
<template>
<v-img :src="image" />
</ template>
<script setup>
import image from "@/assets/image.img"
</ script>